jobs/rocoto/efcs.sh

```diff
@@ -4,9 +4,17 @@ source "${HOMEgfs}/ush/preamble.sh"
 
 ###############################################################
 # Source FV3GFS workflow modules
-. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
-status=$?
-[[ ${status} -ne 0 ]] && exit ${status}
+#. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
+#status=$?
+#[[ ${status} -ne 0 ]] && exit ${status}
+
+set +x
+module use "${HOMEgfs}/sorc/ufs_model.fd/tests"
+module load modules.ufs_model.lua
```

Comment:
If this is a long-term solution, I think we're going to need to copy these modulefiles  to our top level in `link_workflow.sh` to satisfy NCO.

Reply:
This is not a long term solution, but is needed ATM.
We can no longer delay the job-specific loading of the modules.  The JEDI team has its own `load_ufsda_modules.sh` as a workaround, but IMO, a better more sound solution needs to be developed.

Reply:
I think it might be good to give each job/task flexibility to load its own modules. This will allow smooth transition to new libraries to all the workflow components during development stage. Maybe we can have a unified version for opn implementation tags when the code is frozen.

Reply:
It appears that just loading the modules from ufs-weather-model modulefile is not enough for wcoss2.
wcoss2 also requires `cray-pals` that loads the `mpiexec`.
This module is loaded in the run script of the regression test
https://github.com/ufs-community/ufs-weather-model/blob/495c5fd106ff3facb5b4aedbf94f8bba6c1da091/tests/fv3_conf/fv3_qsub.IN_wcoss2#LL20-L20C22
Pity!

Reply:
@junwang-noaa Can we load `cray-pals` in https://github.com/ufs-community/ufs-weather-model/blob/develop/modulefiles/ufs_wcoss2.intel.lua? 
It will save us a mountain of trouble till we have a proper solution?

Reply:
Yes, we can. This will require model version change, I will submit a PR to see if we can get it in as soon as possible.

Reply:
Thank you @junwang-noaa.  We will proceed w/ the hack we have for now.
Let us know when there is an update in the model hash.